src/main/java/ui/components/NavigableListView.java

```diff
@@ -111,25 +114,43 @@ public void restoreSelection() {
                 selectedIndex = Optional.of(nextIndex);
 
                 // The next item will be considered selected
-                onItemSelected.accept(nextIndex);
+                triggerItemSelected(nextIndex);
             }
         }
     }
 
+    /**
+     * Passes the relevant index to the onItemSelected callback variable.
```

Comment:
Can this be rephrased to make it easier to understand? Similar to how we changed the method name?